.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+/*            @erbridge @CJWaszczuk @LBHELewis
+
+# Any changes in a directory starting with `.` in the root of the repository.
+/.*/          @erbridge @CJWaszczuk @LBHMGeorgieva @Duslerke @LBHELewis
+/.*/          @LBHackney-IT/mat-developers @LBHELewis
+
+/docs/        @erbridge @CJWaszczuk @LBHELewis
+/docs/adr/    @erbridge @CJWaszczuk @LBHMGeorgieva @Duslerke @LBHELewis
+/docs/adr/    @LBHackney-IT/mat-developers @LBHELewis
+
+/src/         @erbridge @CJWaszczuk @LBHMGeorgieva @Duslerke @LBHELewis
+/examples/    @erbridge @CJWaszczuk @LBHMGeorgieva @Duslerke @LBHELewis
+/src/         @LBHackney-IT/mat-developers @LBHELewis
+/examples/    @LBHackney-IT/mat-developers @LBHELewis
+
+# We disable these to reduce noise from Dependabot.
+package.json
```

Comment:
I think this can be a lot simpler:

```
# The people in this file are **not** responsible for reviewing this code.
# That falls to the people working on it directly. These people are here to
# explicitly notify them, to keep them informed of what's happening to this
# repository.

# The default, matching to all changes, overriden by any later rules.
*             @LBHackney-IT/mat-developers @LBHELewis

# We disable these to reduce noise from Dependabot.
package.json
package-lock.json
```